website/source/docs/connect/ca/aws.html.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: "docs"
+page_title: "Connect - Certificate Management"
+sidebar_current: "docs-connect-ca-aws"
+description: |-
+  Consul can be used with AWS Certificate Manager Private CA to manage and sign certificates.
+---
+
+# AWS Certificate Manager Private CA as a Connect CA
+
+Consul can be used with [AWS Certificate Manager (ACM) Private Certificate
+Authority
+(CA)](https://aws.amazon.com/certificate-manager/private-certificate-authority/)
+to manage and sign certificates.
+
+-> This page documents the specifics of the AWS ACM Private CA provider.
+Please read the [certificate management overview](/docs/connect/ca.html)
+page first to understand how Consul manages certificates with configurable
+CA providers.
+
+## Requirements
+
+The ACM Private CA Provider needs to be authorized via IAM credentials to
+perform operations. Every Consul Server needs to be running in an environment
+where a suitable IAM configuration is present.
+
+The [standard AWS SDK credential
+locations](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials)
+are used which means that suitable credentials and region configuration need to be present in one of:
+ 1. Environment variables
+ 2. Shared credentials file
+ 3. Via an EC2 instance role
+
+The IAM credential provided must have permission for the following actions:
+
+ - CreateCertificateAuthority - assuming an existing CA is not specified in `existing_arn`
+ - DescribeCertificateAuthority
+ - GetCertificate
+ - IssueCertificate
+
+## Configuration
+
+The ACM Private CA provider is enabled by setting the `ca_provider` to
+`"aws-pca"`. At this time there is only one, optional configuration value.
+
+```hcl
+connect {
+    enabled = true
+    ca_provider = "aws-pca"
+    ca_config {
+      existing_arn = "..."
+    }
+}
+```
+
+ ~> Note that suitable AWS IAM credentials are necessary for the provider to
+ work, however these are not configured in the Consul config which is typically
+ on disk and rely on the [standard AWS SDK configuration
+ locations](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html#specifying-credentials).
+
+The set of configuration options is listed below. The
+first key is the value used in API calls while the second key (after the `/`)
+is used if configuring in an agent configuration file.
```

Comment:
```suggestion
is used if you're adding configuring to the agent's configuration file.
```

Reply:
I copied this from the Vault page so we should maybe update that to match too?